packages/uniforms/src/AutoForm.tsx

```diff
@@ -15,7 +16,10 @@ import { ModelTransformMode, UnknownObject } from './types';
 
 export type AutoFormProps<Model extends UnknownObject> =
   ValidatedQuickFormProps<Model> & {
-    onChangeModel?: (model: Model) => void;
+    onChangeModel?: (
+      model: Model,
+      info: { key: string; value: unknown; previousValue: unknown },
```

Comment:
I don't like the `info` key, what do you think about `context`, `changeContext`, `details`, `changeDetails`, `meta`, `metadata`
```suggestion
      changeContext: { key: string; value: unknown; previousValue: unknown },
```